### DISCUSSION TITLE
[off topic] Shadow when selecting text, on Pradyun's blog

### DISCUSSION THREAD

**Discussion by gakudva** [pradyunsg/furo#120] [Potential Issue] [answered]
Referring to screenshot - when you select a text, the text seems to have an unwanted shadow.  I was not sure if this was intended.  Looked like a potential issue to me.
<img width="1430" alt="shadow in selected" src="https://user-images.githubusercontent.com/2473960/113479534-4d2d4e80-94ad-11eb-98e8-230a53475e4b.png">

**Comment by pradyunsg** [comment, marked as answer]
IIUC, that's from my blog (https://github.com/pradyunsg/pradyunsg.github.io) and not Furo. :)

**Reply by gakudva** [reply]
Thanks for the response!

yes - thats from your blog. I raised it here as I assumed that issue comes from underlying Furo theme.  I checked the other sites that use Furo - they don' have this symptom.  So, its an issue with your blog rather than Furo.  Sorry!

**Reply by pradyunsg** [reply]
No worries. It's intentional FWIW, since the normal link color does not have enough contrast against the highlighted background when you select more than just the link you're hovering over.
